slim.Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM python:3.7-alpine as builder
+LABEL maintainer="Kitware, Inc. <kitware@kitware.com>"
+EXPOSE 8080
+
+# Set environment to support Unicode: http://click.pocoo.org/5/python3/#python-3-surrogate-handling
+ENV LC_ALL=C.UTF-8
+ENV LANG=C.UTF-8
+
+WORKDIR /girder
+
+# avoid psutil GCC dependency by using unofficial large_image wheel
+RUN apk add --no-cache git gcc musl-dev libffi-dev openssl-dev
+
+COPY . ./
+
+RUN pip install \
+    --no-cache-dir \
+    --find-links https://girder.github.io/large_image_wheels \
+    .
+
+FROM python:3.7-alpine as runtime
+
+COPY --from=builder /usr/local/lib/python3.7/site-packages/ /usr/local/lib/python3.7/site-packages/
+COPY --from=girder/girder:latest /usr/share/girder/static/ /usr/local/share/girder/static/
+
+ENTRYPOINT ["python", "-c", "from girder import cli; cli.main()", "serve"]
```

Comment:
Since girder is no longer pip installed in this image, it must be started in this way.  The result is identical, as far as I can tell.